### PR TITLE
Some Tests

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/Http/Requests/StoreProductRequest.php
Comment (Huma-volve) [inline app/Http/Requests/StoreProductRequest.php:1]: 1- make all requests like this please your are making correclty but we need to make all project standard so 
making your request like this 
            'category_id' => [
            'required',
            'exists:categories,id'
            ]

and so on the rest of fileds

File: app/Models/Product.php
Comment (Huma-volve) [inline app/Models/Product.php:1]: using $fillable instead of $guarded

File: routes/api.php
Comment (Huma-volve) [inline routes/api.php:1]: Why not making middleware for auth:sanctum global on all routes that must be authenticated ?
making favorite as apiResource and use only method for destory, store, index

**Review by ghost — COMMENTED** [review]
Please follow the review

**Pull request merged**